### PR TITLE
Fix downloaders tests

### PR DESCRIPTION
**Pull request by arkid15r**
## Purpose/Implementation Notes

Fix tests for `downloader_no_op_tests`:
  - Update requirements.
  - Fix aspera downloader.
  - Migrate to new sra data source.
  - Update docker image.

## Types of changes

- Bugfix (non-breaking change which fixes an issue)

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: workers/keys/asperaweb_id_dsa.openssh
Comment (davidsmejia) [inline workers/keys/asperaweb_id_dsa.openssh:1]: I think that we should have a `README.md` in this folder that explains the origins of this key and it's purpose for future devs.

**Review by davidsmejia — CHANGES_REQUESTED** [review]
LGTM just one documentation request.

File: workers/keys/asperaweb_id_dsa.openssh
Comment (arkid15r) [reply workers/keys/asperaweb_id_dsa.openssh:1]: Done.

File: workers/keys/README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+`asperaweb_id_dsa.openssh` is a publicly available "private" key required by ascp.
+See https://www.ibm.com/docs/en/aspera-on-cloud?topic=ak-fasp-security-aspera-connect-transfer-client-authorized-keys-aspshell
```
Comment (davidsmejia) [inline workers/keys/README.md:2]:
```suggestion
See [IBM's documentation](https://www.ibm.com/docs/en/aspera-on-cloud?topic=ak-fasp-security-aspera-connect-transfer-client-authorized-keys-aspshell)
```

**Review by davidsmejia — APPROVED** [review]
LGTM 🐼

File: workers/keys/README.md (hunk 1 above)
Comment (arkid15r) [reply workers/keys/README.md]: 👍

**Pull request merged**